styleguide/_themes/derek/scss/components/subnav.scss

```diff
@@ -136,7 +136,7 @@
     border-bottom: 0;
     border-left: 2px solid transparent;
     height: auto;
-    margin-top: 0;
+    margin: 0;
```

Comment:
I don't think this solves the problem. The issue itself is not with the `.subnav-item` class, but the `.subnav-noHover` class. If you change the following in `.subnav-noHover`, I think it's much cleaner.
```
    margin: 5px 50px 5px 0;
```
to:
```
    margin: 5px auto;
```